kinder/pkg/cluster/manager/actions/waiter.go

```diff
@@ -54,6 +54,18 @@ func waitForPodsRunning(c *status.Cluster, n *status.Node, wait time.Duration, l
 	return nil
 }
 
+// waitForNodePort waits for a nodePort to become ready
+func waitForNodePort(c *status.Cluster, n *status.Node, wait time.Duration, nodePort string) error {
+	n.Infof("waiting for NodePort to become ready (timeout %s)", wait)
```

Comment:
potentially include nodePort in the message:

```
n.Infof("waiting for NodePort %q to become ready (timeout %s)", nodePort, wait)
```

Reply:
👍